contracts/Oracle/Calculations/Sushiswap.sol

```diff
@@ -112,18 +108,12 @@ contract CalculationsSushiswap {
         uint256 amountIn = 10**uint256(token0.decimals());
         uint256[] memory amountsOut;
 
-        bool fallbackRouterExists = secondaryRouterAddress != zeroAddress;
-        if (fallbackRouterExists) {
-            try primaryRouter.getAmountsOut(amountIn, path) returns (
-                uint256[] memory _amountsOut
-            ) {
-                amountsOut = _amountsOut;
-            } catch {
-                amountsOut = secondaryRouter.getAmountsOut(amountIn, path);
-            }
-        } else {
-            amountsOut = primaryRouter.getAmountsOut(amountIn, path);
+        PriceRouter router = defaultRouter;
+        address routerOverrideAddress = routerOverrides[token0Address];
```

Comment:
Support overrides

Reply:
will uni v3 support the PriceRouter interface, or will we need to deploy a wrapper contract around it?

Reply:
We would need to deploy a wrapper contract for uni v3